web/app/components/header/nav.hbs

```diff
@@ -81,11 +81,13 @@
               "highlighted-new"
             }}
           />
-          <dd.Interactive
-            data-test-user-menu-item="sign-out"
-            {{on "click" this.invalidateSession}}
-            @text="Sign out"
-          />
+          {{#if this.showSignOut}}
+            <dd.Interactive
+              data-test-user-menu-item="sign-out"
+              {{on "click" this.invalidateSession}}
+              @text="Sign out"
+            />
+          {{/if}}
```

Comment:
Why are we removing the logout button? What happens if I `skipGoogleAuth` and try to `invalidateSession`?

Reply:
It doesn't do anything because there's no Ember Simple Auth session to invalidate when we bypass Google auth.

Reply:
Can we have it invalidate with Okta instead? I assume users will still want to sign out.

Reply:
Our supported Okta configuration enforces Okta auth at the load balancer level so there's not going to be a way to sign out unless the user signs out of their Okta session (which will be in Okta, not Hermes).

Reply:
Can we change our supported configuration to allow logout? 😬 I'd really like to keep the log out button. "As a user I want to log out."

Reply:
We chatted about this offline, but I think I found a way to do this and it'll be implemented in another PR soon!